src/_includes/layouts/homepage.html

```diff
@@ -20,7 +20,7 @@
 		<main data-layout="centered-single-column">
 			<h2 class="events-title">Upcoming Events</h2>
 			<section id="upcoming-events-section" class="events-section">
-				{% for event in collections.upcomingEvents %} {{ event | log }}
```

Comment:
Thank you so much for nuking this `log`! 😅

Reply:
oh shoot! actually didn't mean to commit that. but it *does* make the reloads faster